hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedMetadata.java

```diff
@@ -258,6 +259,7 @@ public void testMetadataTableBootstrap(HoodieTableType tableType, boolean addRol
     validateMetadata(testTable, true);
   }
 
+  @Disabled("HUDI-9221")
```

Comment:
we might need to investigate and re-enable these tests.

Reply:
It is tracked in HUDI-9221